From 27b8d5690f0b83ffc6032c55a5ffa3701848d7ea Mon Sep 17 00:00:00 2001
From: Rinigus <rinigus.git@gmail.com>
Date: Thu, 13 Nov 2025 18:42:44 +0000
Subject: [PATCH] Take into account env QTWEBENGINE_CHROMIUM_FLAGS

---
 src/main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/main.cpp b/src/main.cpp
index 62bc4ff5..1ed61edf 100644
--- a/src/main.cpp
+++ b/src/main.cpp
@@ -48,7 +48,7 @@ Q_DECL_EXPORT int main(int argc, char *argv[])
 
     // Setup QtWebEngine
     qputenv("QTWEBENGINE_DIALOG_SET", "QtQuickControls2");
-    QString chromiumFlags;
+    QString chromiumFlags = QString::fromLocal8Bit(qgetenv("QTWEBENGINE_CHROMIUM_FLAGS"));
     if (AngelfishSettings::self()->webSmoothScrollingEnabled()) {
         chromiumFlags.append(QStringLiteral(" --enable-smooth-scrolling"));
     }
-- 
2.43.7

